{"uuid": "84b37244-5bb2-42cb-b644-6e7b5ce0c3d3", "children": ["9dad4660-b743-436a-894c-71c89ee7200e"], "befores": [{"name": "in_browser", "status": "passed", "start": 1760898089648, "stop": 1760898090837}], "afters": [{"name": "in_browser::0", "status": "broken", "statusDetails": {"message": "ConnectionResetError: [WinError 10054] Удаленный хост принудительно разорвал существующее подключение\n", "trace": "  File \"C:\\Users\\Denzel\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\allure_commons\\_allure.py\", line 231, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Denzel\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 938, in _teardown_yield_fixture\n    next(it)\n    ~~~~^^^^\n  File \"C:\\Users\\Denzel\\PycharmProjects\\qa_huru_hw_10_jenkins\\tests\\test\\conftest.py\", line 33, in in_browser\n    browser.quit()\n    ~~~~~~~~~~~~^^\n  File \"C:\\Users\\Denzel\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selene\\core\\_browser.py\", line 143, in quit\n    self.driver.quit()\n    ~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Denzel\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 224, in quit\n    self.service.stop()\n    ~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Denzel\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 152, in stop\n    self.send_remote_shutdown_command()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Denzel\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 132, in send_remote_shutdown_command\n    request.urlopen(f\"{self.service_url}/shutdown\")\n    ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Denzel\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\urllib\\request.py\", line 189, in urlopen\n    return opener.open(url, data, timeout)\n           ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Denzel\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\urllib\\request.py\", line 489, in open\n    response = self._open(req, data)\n  File \"C:\\Users\\Denzel\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\urllib\\request.py\", line 506, in _open\n    result = self._call_chain(self.handle_open, protocol, protocol +\n                              '_open', req)\n  File \"C:\\Users\\Denzel\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\urllib\\request.py\", line 466, in _call_chain\n    result = func(*args)\n  File \"C:\\Users\\Denzel\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\urllib\\request.py\", line 1348, in http_open\n    return self.do_open(http.client.HTTPConnection, req)\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Denzel\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\urllib\\request.py\", line 1323, in do_open\n    r = h.getresponse()\n  File \"C:\\Users\\Denzel\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\http\\client.py\", line 1430, in getresponse\n    response.begin()\n    ~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Denzel\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\http\\client.py\", line 350, in begin\n    self.headers = self.msg = parse_headers(self.fp)\n                              ~~~~~~~~~~~~~^^^^^^^^^\n  File \"C:\\Users\\Denzel\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\http\\client.py\", line 248, in parse_headers\n    headers = _read_headers(fp)\n  File \"C:\\Users\\Denzel\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\http\\client.py\", line 221, in _read_headers\n    line = fp.readline(_MAXLINE + 1)\n  File \"C:\\Users\\Denzel\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\socket.py\", line 719, in readinto\n    return self._sock.recv_into(b)\n           ~~~~~~~~~~~~~~~~~~~~^^^\n"}, "start": 1760898102994, "stop": 1760898103368}], "start": 1760898089648, "stop": 1760898103368}